eventservice/api/utils/group_auth.py

```diff
@@ -61,6 +61,18 @@ def get_group_memberships():
 
 # This util method checks if a user has admin permissions in a group event
 def check_group_event_admin_access(event, group_memberships):
+    if event and event.get('groupIds'):
+        if len(event.get('groupIds')) != len(group_memberships):
```

Comment:
Hi, @bingzhang, Could you please look into this check here when you get a chance? We may not need this as the groupIds in an event can be [A, B] while the groups that I'm an admin of can be [A, B, C, D]. So, the lengths won't match, but I should have permission to create an event with these group settings. Please let me know if you have any questions.

Also, Could you please do some more testing with creating different events once you make changes? Thanks.

Reply:
Resolving this conversation as the code has been updated here https://github.com/rokwire/rokwire-building-blocks-api/pull/1024/commits/af626f84714dbda7b27a8bf2e3288eeab5b353bd. Thanks.